packages/cli/src/index.ts

```diff
@@ -125,39 +126,89 @@ cli.command('deploy', 'deploy the app to the marketplace')
         });
     });
 
-cli.command('create [appName]', 'create a new marketplace app').action(async (appName: string) => {
-    const { promptedAppName, stylingFramework } = await prompts([
-        {
-            type: 'text',
-            name: 'promptedAppName',
-            message: 'Enter your app name',
-            initial: appName || 'my-frontify-app',
-            validate: (value: string) => {
-                if (value.trim() === '') {
-                    return 'You need to enter an app name.';
-                }
-
-                return isValidName(value);
-            },
-        },
-        {
-            type: 'select',
-            name: 'stylingFramework',
-            message: 'Choose a styling framework',
-            choices: [
-                { title: 'Tailwind', value: 'tailwind' },
-                { title: 'CSS Modules', value: 'css-modules' },
-                { title: 'None', value: 'css' },
-            ],
-        },
-    ]);
-
-    if (!promptedAppName || !stylingFramework) {
-        exit(0);
-    }
-
-    createNewContentBlock(promptedAppName, stylingFramework);
-});
+cli.command('create [appName]', 'create a new marketplace app')
+    .option('-e, --experimental', 'set experimental flag')
+    .action(async (appName: string, options) => {
+        if (options.experimental) {
+            const { promptedAppName, stylingFramework, appType } = await prompts([
+                {
+                    type: 'text',
+                    name: 'promptedAppName',
+                    message: 'Enter your app name',
+                    initial: appName || 'my-frontify-app',
+                    validate: (value: string) => {
+                        if (value.trim() === '') {
+                            return 'You need to enter an app name.';
+                        }
+
+                        return isValidName(value);
+                    },
+                },
+                {
+                    type: 'select',
```

Comment:
Perhaps we can refactor it a bit and get rid of the (almost) duplicate if-else blocks. E.g., this `appType` prompt could be conditionally skipped:
`type: (options.experimental) ? 'select' : null,`

Reply:
Or perhaps not 🤔 This could make it messier and harder to clean up when we're ready to roll out the "experimental" approach..